var/spack/repos/builtin/packages/libimobiledevice/package.py

```diff
@@ -0,0 +1,34 @@
+# Copyright 2013-2019 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class Libimobiledevice(AutotoolsPackage):
+    """Library to communicate with iOS devices natively."""
+
+    homepage = "https://www.libimobiledevice.org/"
+    url      = "https://www.libimobiledevice.org/downloads/libimobiledevice-1.2.0.tar.bz2"
+    git      = "https://git.libimobiledevice.org/libimobiledevice.git"
```

Comment:
Should we consider using Github for `url` and `git`? See https://github.com/libimobiledevice/libimobiledevice

Reply:
Nah, GitHub tarball doesn't come with a `configure` script